src/posts/keymaps/naginata.md

```diff
@@ -0,0 +1,24 @@
+---
+author: "Toshihiko Ooka (大岡俊彦)"
+baseLayouts: [Naginata]
+firmwares: [QMK, ZMK, Karabiner-Elements]
+hasHomeRowMods: false
+hasLetterOnThumb: false
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: true
+isTapDanceEnabled: false
+keybindings: # [Vim, Emacs, Kakoune, Graphics/CAD, TWM, Spreadsheets, Gaming]
+keyboard: # Kyria or Corne or Dactyl Manuform 5x6 or ...
+keyCount: 31
+keymapImage: https://oookaworks.up.seesaa.net/image/E89699E58880E5BC8Fv16E6A0BCE5AD90-thumbnail2.jpg
+keymapUrl: https://github.com/eswai/qmk_userspace/tree/main/users/naginata_v16
+languages: [Japanese]
+layerCount: 3
+OS: [Windows, MacOS, Linux]
+stagger: ortholinear
+summary: "Kana-based Japanese system for ergonomic keyboards with thumbs-keys, or traditional keyboards with spacebar used as shift. Dakuten (or handakuten) are combos with other hand's index-finger home (or bottom) row. Combos with ya/yu/yo give compound kana (yōon, at most three keys). Punctuation, navigation, and macros are on layers. Currently v16 (2025). Multiple community implementations exist."
```

Comment:
```suggestion
summary: "Kana-based Japanese system for ergonomic keyboards with thumbs-keys, or traditional keyboards with spacebar used as shift. Dakuten (or handakuten) are combos with other hand's index-finger home (or bottom) row. Combos with ya/yu/yo give compound kana (yōon, at most three keys). Punctuation, navigation, and macros are on layers. Multiple community implementations exist."
```

What is "index-finger home row" supposed to mean? I can understand index-finger home key but row? Do you mean the resting key and the neighboring key requiring a lateral movement of the index finger?